starter/src/index.html

```diff
@@ -4,30 +4,18 @@
 		<!-- The base href is defined through the Webpack configuration and can be customized through the angular.json file -->
 		<!-- necessary for HTML5 History API which adds support for pretty URLs (see app.ts) -->
 		<meta charset="utf-8" />
-		<title><%= htmlWebpackPlugin.options.starkAppMetadata.name %></title>
 
-		<meta name="description" content="<%= htmlWebpackPlugin.options.starkAppMetadata.description %>" />
-
-		<% if (webpackConfig.htmlElements.headTags) { %>
-		<!--Configured Head Tags  -->
-		<%= webpackConfig.htmlElements.headTags %> <% } %>
+		<title><%= .starkAppMetadata.name %></title>
```

Comment:
There is a missing word here, it should be: 

```html
<%= starkOptions.starkAppMetadata.name %>
```

Reply:
Opps solved :-)